src/features/applicationView/ApplicationView.tsx

```diff
@@ -1,27 +1,26 @@
 import React from 'react';
 import classNames from 'classnames';
 import { useTranslation } from 'react-i18next';
-import { Notification } from 'hds-react';
 import { PureQueryOptions } from 'apollo-client';
 
 import styles from './applicationView.module.scss';
 import Card from '../../common/card/Card';
 import CardBody from '../../common/cardBody/CardBody';
 import ApplicationDetails, { ApplicationDetailsProps } from '../../common/applicationDetails/ApplicationDetails';
 import CardHeader from '../../common/cardHeader/CardHeader';
-import Table, { Column } from '../../common/table/Table';
-import CustomersTableTools, { CustomersTableToolsProps } from './customersTableTools/CustomersTableTools';
 import Text from '../../common/text/Text';
 import { formatDate } from '../../common/utils/format';
 import Chip from '../../common/chip/Chip';
 import { APPLICATION_STATUS } from '../../common/utils/consonants';
 import CustomerProfileCard, { CustomerProfileCardProps } from '../../common/customerProfileCard/CustomerProfileCard';
 import OfferCard, { OfferCardProps } from './offerCard/OfferCard';
 import { OrganizationType } from '../../@types/__generated__/globalTypes';
-import Pagination, { PaginationProps } from '../../common/pagination/Pagination';
 import PageTitle from '../../common/pageTitle/PageTitle';
 import PageContent from '../../common/pageContent/PageContent';
 import ActionHistoryCard from '../../common/actionHistoryCard/ActionHistoryCard';
+import SimilarCustomersTableContainer, {
```

Comment:
Should we move the similar customers stuff right under features folder (`features/simarCustomers`). It starts to look like a feature with own queries and mutations. Should the feature be called something like "linkCustomerTable" or "linkCustomer", what do you think?

Reply:
Makes sense, I did also think about it. How would `linkApplicationToCustomer` sound?

`offerCard/editForm` also has its own queries and mutations, would it make sense to move that one as well?

Reply:
The name you suggested sounds great 👍 
and I think moving "offerCard/editForm" would make sense, if you want to do it in this PR.

Reply:
The forms could be moved later, there are several other "fully featured" forms under other features as well